drivers/accel/adxl355/iio_adxl355.c

```diff
@@ -195,14 +195,6 @@ static struct scan_type adxl355_iio_accel_scan_type = {
 	.is_big_endian = false
 };
 
-static struct scan_type adxl355_iio_temp_scan_type = {
```

Comment:
I actually made these changes to comply with the Linux driver which does not use the temperature in the plot diagram.
This also makes sense when using triggers because the DATA_RDY trigger actually refers only to acceleration data.